eth2/types/src/beacon_state.rs

```diff
@@ -764,6 +768,20 @@ impl<T: EthSpec> BeaconState<T> {
         })
     }
 
+    /// Get the number of outstanding deposits.
+    ///
+    /// Returns `Err` if the state is invalid.
+    pub fn get_outstanding_deposit_len(&self) -> Result<u64, Error> {
+        if let Some(len) = self.eth1_data.deposit_count.checked_sub(self.eth1_deposit_index) {
```

Comment:
Perhaps this is a good fit for (excuse lazy indenting/formatting):

```rust
self.eth1_data
    .deposit_count
    .checked_sub(self.eth1_deposit_index)
    .ok_or_else(|| {
          Err(Error::InvalidDepositState {
              deposit_count: self.eth1_data.deposit_count,
              deposit_index: self.eth1_deposit_index,
          })
    })
```

What do you think? :)

Reply:
It's cool, thank you!